app/views/dashboard/_new_hosts_widget.html.erb

```diff
@@ -0,0 +1,11 @@
+<% hosts = @data.hosts.includes(:operatingsystem).preload(:owner).order(:created_at => :desc).limit(8) %>
```

Comment:
For the record, rails throws a `ActiveRecord::EagerLoadPolymorphicError: Cannot eagerly load the polymorphic association :owner` error.